apis/apps/v1alpha1/cloneset_types.go

```diff
@@ -75,11 +75,30 @@ type CloneSetSpec struct {
 	Lifecycle *appspub.Lifecycle `json:"lifecycle,omitempty"`
 }
 
+// CloneSetDeletePriority defines the relationship among these conditions are ANDed
+type CloneSetDeletePriority struct {
+	// PodNames is the assemble of the pods’ names
+	PodNames []string `json:"podNames,omitempty"`
+	// matchLabels is a map of {key,value} pairs. A single {key,value} in the matchLabels
+	// map is equivalent to an element of matchExpressions, whose key field is "key", the
+	// operator is "In", and the values array contains only "value". The requirements are ANDed.
+	// +optional
+	MatchLabels map[string]string `json:"matchLabels,omitempty" protobuf:"bytes,1,rep,name=matchLabels"`
```

Comment:
please use LabelSelector instead of a map， one may need MatchExpressions instead of MatchLabels